tests/link/pytorch/test_basic.py

```diff
@@ -235,3 +235,20 @@ def test_arange():
         FunctionGraph([start, stop, step], [out]),
         [np.array(1), np.array(10), np.array(2)],
     )
+
+
+def test_eye():
+    N = scalar("N", dtype="int64")
+    M = scalar("M", dtype="int64")
+    k = scalar("k", dtype="int64")
+
+    out = eye(N, M, k, dtype="float32")
+
+    trange = range(1, 6)
+
+    fn = function([N, M, k], out, mode=pytorch_mode)
+
+    for _N in trange:
```

Comment:
I would personally prefer these test cases over `N`, `M`, and `k` to be parameterized, but I'll defer to @ricardoV94

Reply:
I would not parametrize when we can reuse the same function to test different values. Should be a much faster test that way

Reply:
That's exactly the use-case for a fixture :(

Reply:
It is but I find fixtures make it harder to read a test. Anyway there's a point at which parametrize doesn't make sense. Say you have 50 lines of setup code and then want to test a function on 2 inputs at line 51. I wouldn't use parametrize in that case nor a fixture. 

I would just call the function twice.

Obviously the point at which one makes sense vs the other is ambiguous. I am just expressing my opinion and happy for it to be ignored. I suggest to go with his preference here

Reply:
In my experience parametrization can also make test logs unnecessarily verbose. I would leave it as it is but I don't have a strong opinion about it neither. Happy to change it if required.

Reply:
Can stay then, there's still the question of testing default vs manual dtype discussed above